docs/dev/android/setup-android-build-environment.md

```diff
@@ -17,19 +17,24 @@ At the end of this process you should have the following environment variables s
 
 - `ANDROID_HOME`
 - `NDK_HOME`
+- `JAVA_HOME`
 
 The easiest way to install all the dependencies (and automatically
 handle updates), is by using [Android Studio](https://developer.android.com/studio/index.html).
-Once this is installed, the following dependencies can be installed through it:
+Once this is installed, it must be run and the glean project opened to complete initial setup.
```

Comment:
Glean?

Reply:
Well, it's unclear. It's the "glean" project when you're opening it and it has glean-core and glean-sample-app subprojects within it... but sure, we can capitalize it here.